src/graphql/outlier.rs

```diff
@@ -342,6 +342,44 @@ impl From<PreserveOutliersInput> for review_database::OutlierInfoKey {
     }
 }
 
+#[derive(Debug, Clone, Serialize)]
+pub struct PreserveOutlierOutput {
```

Comment:
Thank you for the changes! The update to the `preserveOutliers` mutation looks good. However, I noticed that the output object is named `PreserveOutlierOutput`, while the input object is named `PreserveOutliersInput`. For consistency, I suggest renaming the output object to `PreserveOutliersOutput`. This will ensure that both the input and output objects follow a similar naming pattern.

Once the output object is renamed, it is okay to merge the PR.

Reply:
Thank you for the comments.
The modification has been completed.